src/shogun/features/CombinedFeatures.cpp

```diff
@@ -271,12 +270,12 @@ void CCombinedFeatures::remove_subset()
 
 	subset_changed_post();
 	SG_UNREF(processed);
-	SG_DEBUG("leaving %s::remove_subset()\n", get_name())
+	SG_DEBUG("leaving remove_subset()\n")
```

Comment:
Actually, I just realise: Just remove all the "Entering" and "Leaving" debug messages, they are pointless